modules/nw-operator-cr.adoc

```diff
@@ -120,10 +120,11 @@ configuration.
 are `Multitenant`, `Subnet`, or `NetworkPolicy`. The default value is
 `NetworkPolicy`.
 
-<4> The maximum transmission unit (MTU) for the VXLAN overlay network. This
-value is normally configured automatically, but if the nodes in your cluster do
-not all use the same MTU, then you must set this explicitly to 50 less than the
-smallest node MTU value.
+<4> The maximum transmission unit (MTU) for the VXLAN overlay network. This is detected automatically based on the MTU of the primary network interface. You do not normally need to override the detected MTU.
++
+If the auto-detected value is not what you expected it to be, confirm that the MTU on the primary network interface on your nodes is correct. You cannot use this option to change the MTU value of the primary network interface on the nodes.
++
+If your cluster requires different MTU values for different nodes, then you must set this value to `50` less than the lowest MTU value in your cluster. For example, if some nodes in your cluster have an MTU of `9001`, and some have an MTU of `1500`, then you must set this value to `1450`.
```

Comment:
```suggestion
If your cluster requires different MTU values for different nodes, you must set this value to `50` less than the lowest MTU value in your cluster. For example, if some nodes in your cluster have an MTU of `9001`, and some have an MTU of `1500`, you must set this value to `1450`.
```
According to ISG:
>! if ... then conjunction
Avoid using "then" to introduce an independent clause that follows an "if" clause. For example, write "If you set a password, access is restricted", not "If you set a password, then access is restricted".